### ISSUE TITLE
some csv bugs?

### ISSUE THREAD

**Issue by moveforever** [apache/incubator-mxnet#4546]
Bug1:
upgrade mxnet to 0.91, csv can't traversal all lines, only traversal part lines.
For example, a csv given as listed:
```
1,1
2,2
3,3
4,4
5,5
6,6
7,7
8,8
9,9
10,10
```
when i use CSVIter traverse all lines, it only traverse these lines as followed:
```
[[ 1.  1.]
 [ 2.  2.]
 [ 0.  0.]]
```

**Comment by moveforever** [comment]
Bug2:
what strange phonemenon!
I have a csv file of ten millions instance, and i make use of these instances to train neural network. I regard CSVIter as iteration of network's inputs. when the same program run in different machines, network's validation's loss are small in one machine and on the contrary another network's validation's loss are larger.
But when i self define csv, the program run in different machines, and the loss are both small.
So the phenomenon above indicates CSVIter exist some bugs.

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
